src/lib/InstantSearch.ts

```diff
@@ -321,15 +322,11 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
     }
 
     if (this.routing) {
-      const routingManager = new RoutingManager({
+      this.routingManager = new RoutingManager({
```

Comment:
```suggestion
      this._routingManager = new RoutingManager({
```
We aren't consistent with public/private yet, but we want to keep currently hidden things hidden